server/server.js

```diff
@@ -9,6 +9,7 @@ import bodyParser from 'body-parser';
 import config from './config';
 import httpsServer from './https_server/https_server';
 import redirectServer from './https_server/redirect_server';
+import dbConnect from './db/db_connect';
```

Comment:
Do we really need this wrapper?
I think you can just use mongoose.connect() directly.
Also, please use promise version instead of old style version.
http://mongoosejs.com/docs/connections.html